### ISSUE TITLE
Thanks for ghostToHugo!

### ISSUE THREAD

**Issue by mceachen** [jbarone/ghostToHugo#27]
Thanks for making this project! :100: :1st_place_medal: 

I had a couple trivial things you might want to consider:

* Ghost 3.0 exports dates as ISO, so `-d "2006-01-02T15:04:05.000Z"` might be a better default for `--dateformat` now.

* You point to herring-cove as a good default theme, but that theme has been abandoned.

**Comment by jbarone** [comment]
Hello @mceachen. I appreciate the feedback.

- In my recent work with my rewrite, I discovered that Ghost doesn't actually have a default date export. It appears to be more based on the backend DB that is supporting Ghost. Now that you mention it though, me rewrite and change of the default was based on a sqlite backend. I'll need to investigate further and see what exports look like coming from ghost.org.
- The theme recommendation is just a quick get up and running, that honestly, I ripped directly from Hugo's `import jekyll` command. But if you have a simple, no config theme you might recommend, I'm open.
